### PR TITLE
[CSS] Fix some errors

### PR DESCRIPTION
**Pull request by prudho**
Following #274.

Most of theses error are wrong css properties or values

### CODE REVIEW

File: src/definitions/elements/label.less [hunk 1]
```diff
@@ -1181,7 +1180,7 @@ a.ui.basic.label:hover {
 .ui.pointing.label:before {
   background-color: inherit;
   background-image: inherit;
-  border-width: none;
+  border-width: 0px;
```
Comment (hammy2899) [inline src/definitions/elements/label.less]: Could you change the likes of these to be `0` instead of having a floating `px` at the end

File: src/definitions/elements/step.less [hunk 2]
```diff
@@ -153,9 +153,6 @@
   flex: 0 1 auto;
   align-self: @iconAlign;
 }
-.ui.steps .step > .icon ~ .content {
```
Comment (hammy2899) [inline src/definitions/elements/step.less]: Why was this removed?

Comment (prudho) [reply src/definitions/elements/step.less]: The rule above was a duplicate as `flex-grow` is a part of the `flex` property.

Comment (hammy2899) [reply src/definitions/elements/step.less]: Ok cool

File: src/definitions/elements/label.less (hunk 1 above)
Comment (prudho) [reply src/definitions/elements/label.less]: Just this one or all of them ?

Comment (hammy2899) [reply src/definitions/elements/label.less]: All of them, if you want you could do it for all files.
